engine/src/flutter/shell/platform/android/platform_view_android_delegate/platform_view_android_delegate.cc

```diff
@@ -37,6 +38,18 @@ void putStringAttributesIntoBuffer(
   }
 }
 
+void putStringIntoBuffer(const std::string& string,
+                         int32_t* buffer_int32,
+                         size_t& position,
```

Comment:
nit: It's usually more clear to pass in a variable as a pointer instead of a reference when you are going to modify it.  That makes it more clear from the callsite that it is being modified.

`foo(variable)` vs `foo(&variable)`